jvm/src/test/scala/uk/gov/homeoffice/drt/db/UserFeedbackDaoSpec.scala

```diff
@@ -0,0 +1,87 @@
+package uk.gov.homeoffice.drt.db
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.Sink
+import org.specs2.mutable.Specification
+import slick.dbio.DBIO
+import slick.jdbc.PostgresProfile.api._
+
+import java.sql.Timestamp
+import java.time.Instant
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.DurationInt
+
+class UserFeedbackDaoSpec extends Specification {
```

Comment:
Again, you could use with Before so you don't have to manually call setup for each test